pylon/pylon_stream_fake.py

```
@@ -6,6 +6,7 @@
 import sys
 import time
 import pathlib
+import logging
```

Ruff

`logging` imported but unused

Remove unused import: `logging`

(F401)